test/test-ipcctl-notifications.js

```diff
@@ -115,7 +115,7 @@ function scaleDown(cb) {
     ee.once('exit', function(n) {
       assert(n.id > 0, 'Worker ID should be present');
       assert(n.pid > 0, 'Worker pid should be present');
-      assert(n.pst > 0, 'Worder start time should be present');
+      assert(n.pst > 0, 'Worker start time should be present');
```

Comment:
I would do in a seperate commit... or this is the kind of thing that can be comitted directly to master.